pkg/progress/action.go

```diff
@@ -0,0 +1,194 @@
+package progress
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"time"
+
+	crv1alpha1 "github.com/kanisterio/kanister/pkg/apis/cr/v1alpha1"
+	"github.com/kanisterio/kanister/pkg/client/clientset/versioned"
+	"github.com/kanisterio/kanister/pkg/field"
+	"github.com/kanisterio/kanister/pkg/log"
```

Comment:
```suggestion
	func "github.com/kanisterio/kanister/pkg/function"
	"github.com/kanisterio/kanister/pkg/log"
```

Reply:
`func` is a keyword so it won't work.

Reply:
Oops! I guess the whole word can be used then 👍🏼